app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">palFinder</string>
+
+    <!-- Strings used for fragments for navigation -->
+    <string name="previous">Précédent</string>
+
+    <!-- Strings used for image loading -->
+    <string name="image_load_failed">Désolé, nous n\'avons pas pu charger l\'image que vous avez choisi.</string>
+
+    <string name="tag_element_text">tag</string>
+    <string name="add_tags">Ajouter des tags</string>
+    <string name="add_tags_click">Cliquer pour sélectionner des tags</string>
+
+
+    <string name="profile_element">nom</string>
+
+    <!-- Strings used for login activity-->
+
+    <string name="search_no_location">Veuillez ajouter une localisation.</string>
+    <string name="search_location_not_found">Localisation introuvable.</string>
+
+    <!-- Strings used for no account user-->
+    <string name="skip_connection"> Sauter la connection </string>
+    <string name="cancel"> Annuler</string>
+    <string name="continu"> Continuer</string>
+    <string name="login"> Login</string>
+    <string name="no_account_warning">Sans vous connectez, vous ne pourrez pas rejoindre de rencontre.</string>
+    <string name="no_account_create"> Vous devez être connecté pour créer une rencontre.</string>
```

Comment:
Is the space at the beginning of the sentence really useful?

Reply:
corrected